make/migrations/postgresql/0001_initial_schema.up.sql

```diff
@@ -285,7 +285,7 @@ UNIQUE(namespace)
 create table properties (
  id SERIAL NOT NULL,
  k varchar(64) NOT NULL,
- v varchar(128) NOT NULL,
+ v varchar(256) NOT NULL,
```

Comment:
put the `ALTER TABLE` statement in `0002_xxxx.up.sql` so the migrator in adminserver can handle the schema upgrade.